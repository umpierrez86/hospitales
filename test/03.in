leer_instancia
2 3
1 1
0 1 2
0 1 2
1 0
0 1
1 0
imprimir_instancia
asignar_residencias
imprimir_asignacion
Fin