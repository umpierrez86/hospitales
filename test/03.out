1>2>Cantidad hospitales: 2
Cupos hospitales: 1 1 
Cantidad estudiantes: 3
Preferencias hospitales
Preferencias hospital h0: 0 1 2 
Preferencias hospital h1: 0 1 2 
Preferencias estudiantes
Preferencias estudiante e0: 1 0 
Preferencias estudiante e1: 0 1 
Preferencias estudiante e2: 1 0 
3>4>(0, 1) (1, 0) (2, libre) 
5>Fin.
